src/test/java/com/researchspace/webapp/integrations/digitalcommonsdata/Base64Util.java

```diff
@@ -0,0 +1,43 @@
+package com.researchspace.webapp.integrations.digitalcommonsdata;
+
+import java.util.UUID;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.ContentDisposition;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+public class Base64Util {
```

Comment:
as this is only used in test, maybe rename to `Base64TestUtil`?

Reply:
renamed